spring-web/src/main/java/org/springframework/web/cors/DefaultCorsProcessor.java

```diff
@@ -83,9 +83,14 @@ public boolean processRequest(@Nullable CorsConfiguration config, HttpServletReq
 			response.addHeader(HttpHeaders.VARY, HttpHeaders.ACCESS_CONTROL_REQUEST_HEADERS);
 		}
 
-		if (!CorsUtils.isCorsRequest(request)) {
+		IsCorsRequestResult isCorsRequestResult = CorsUtils.isCorsRequest(request);
```

Comment:
Note: there is another caller of this function: `org.springframework.web.cors.reactive.DefaultCoreProcessor`.